src/clients/client.ts

```diff
@@ -189,6 +191,40 @@ class Client {
     return await this.httpRequest.post(url, params)
   }
 
+  ///
+  /// Multi Search
+  ///
+
+  /**
+   * Perform multiple search queries.
+   *
+   * It is possible to make multiple search queries on the same index or on
+   * different ones
+   *
+   * @example
+   *
+   * ```ts
+   * client.multiSearch({
+   *   queries: [
+   *     { indexUid: 'movies', q: 'wonder' },
+   *     { indexUid: 'books', q: 'flower' },
+   *   ],
+   * })
+   * ```
+   *
+   * @param queries - Search queries
+   * @param config - Additional request configuration options
+   * @returns Promise containing the search responses
+   */
+  async multiSearch<T extends Record<string, any> = Record<string, any>>(
+    queries?: MultiSearchParams,
+    config?: Partial<Request>
```

Comment:
What is the point of this argument? `config?: Partial<Request>`

Reply:
If user wants to provide [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) options but not all of them (that's the `Partial` part).

It is used during the HTTP request:
```
 return await this.request({
      method: 'GET',
      url,
      params,
      config, // Here
    })
```